### PR TITLE
Addition to data analyses in python lesson

### PR DESCRIPTION
**Pull request by joelostblom**
Added resources to learn more and a couple of minor updates.

### CODE REVIEW

File: lessons/python/intro-data-analysis/from-spreadsheets-to-pandas-extended.ipynb [hunk 1]
```diff
@@ -2107,6 +2124,21 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Resources to learn more <a class=\"anchor\" id=\"resources-to-learn-more\"></a>\n",
+    "\n",
+    "The documentation for these packages are great resources to learn more.\n",
+    "\n",
+    "- [Introduction to `pandas`](http://pandas.pydata.org/pandas-docs/stable/10min.html)\n",
```
Comment (QuLogic) [inline lessons/python/intro-data-analysis/from-spreadsheets-to-pandas-extended.ipynb]: https on all these links.

Comment (joelostblom) [reply lessons/python/intro-data-analysis/from-spreadsheets-to-pandas-extended.ipynb]: Classic! I just installed https everywhere, hopefully that will rid the problem for the future.

**Review by QuLogic — APPROVED** [review]

**Review by QuLogic — CHANGES_REQUESTED** [review]
Changed my mind; you changed the description from molten to long, but didn't rename the variables to match what was done in the lecture.

**Comment by joelostblom** [comment]
Good call

**Review by QuLogic — APPROVED** [review]

**Pull request merged**